src/main/java/uk/gov/justice/hmpps/prison/api/resource/BookingResource.java

```diff
@@ -651,6 +651,17 @@ public List<OffenceHistoryDetail> getOffenceHistory(@PathVariable("offenderNo")
         return bookingService.getOffenceHistory(offenderNo, convictionsOnly);
     }
 
+    @ApiResponses({
+        @ApiResponse(responseCode = "200", description = "OK"),
+        @ApiResponse(responseCode = "400", description = "Invalid request.", content = {@Content(mediaType = "application/json", schema = @Schema(implementation = ErrorResponse.class))}),
+        @ApiResponse(responseCode = "500", description = "Unrecoverable error occurred whilst processing request.", content = {@Content(mediaType = "application/json", schema = @Schema(implementation = ErrorResponse.class))})})
+    @Operation(summary = "Offence histories.", description = "Offence histories for a set of booking ids.")
+    @PostMapping("/offenceHistory")
```

Comment:
think general naming convention is `kebab-case`

Reply:
updated